src/bun.js/bindings/BunProcess.cpp

```diff
@@ -2294,12 +2295,18 @@ JSC_DEFINE_HOST_FUNCTION(Process_functionCpuUsage,
     if (callFrame->argumentCount() > 0) {
         JSValue comparatorValue = callFrame->argument(0);
         if (!comparatorValue.isUndefined()) {
-            if (UNLIKELY(!comparatorValue.isObject())) {
+            if (UNLIKELY(!comparatorValue.isCell())) {
```

Comment:
nit: we could probably drop this check entirely, and rely on `getObject()` returning null